### PR TITLE
Add docs for adding execution nodes and custom CA

### PR DESCRIPTION
**Pull request by fosterseth**
related https://github.com/ansible/awx-operator/issues/1059
##### SUMMARY

Instructions on adding execution node capacity to AWX deployments.

Also, instructions about adding a custom CA secret.

TODO: if we change CA secret to type TLS, we should amend this doc

##### ISSUE TYPE

 - New or Enhanced Feature

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1200,6 +1202,31 @@ Then install the new AWX Operator by following the instructions in [Basic Instal
...
+#### Custom Receptor CA
+The control nodes on the K8S cluster will communicate with execution nodes via mutual TLS TCP connections, running via Receptor.
+Execution nodes will verify incoming connections by ensuring the x509 certificate was issued by a trusted Certificate Authority (CA).
+
+A user may wish to provide their own CA for this validation. If no CA is provided, AWX Operator will automatically generate one using OpenSSL.
+
+```bash
+kubectl create secret generic awx-demo-receptor-ca \
+   --from-file=receptor-ca.crt=./receptor-ca.crt --from-file=receptor-ca.key=./receptor-ca.key
+```
```
Comment (TheRealHaoLiu) [inline README.md]: i got a PR in to turn this into a tls secret

Comment (TheRealHaoLiu) [reply README.md]:
```
kubectl create secret tls awx-receptor-ca --cert=path/to/tls.cert --key=path/to/tls.key
```

**Comment by fosterseth** [comment]
made change to be compatible with https://github.com/ansible/awx-operator/pull/1063

File: README.md [hunk 2]
```diff
@@ -1200,6 +1202,33 @@ Then install the new AWX Operator by following the instructions in [Basic Instal
...
+kubectl create secret tls awx-demo-receptor-ca \
+   --cert=/path/to/ca.crt --key=/path/to/ca.key
+```
+
+The secret should be named `{AWX Custom Resource name}-receptor-ca`. In the above the AWX CR name is "awx-demo". Please replace "awx-demo" with your AWX Custom Resource name.
+
+If this secret is created after AWX is deployed, run the following to restart the deployment,
+
+```bash
+kubectl rollout restart deployment awx
```
Comment (TheRealHaoLiu) [inline README.md]: If awx instance name is awx-demo than the deployment I think also is awx-demo?

Comment (fosterseth) [reply README.md]: you're right

**Review by TheRealHaoLiu — APPROVED** [review]

**Pull request merged**